code/modules/projectiles/guns/energy/special.dm

```diff
@@ -171,7 +172,7 @@
 			to_chat(user, "<span class='notice'>You try to insert [I] into [src], but it's fully charged.</span>") //my cell is round and full
 			return
 		I.use(1)
-		cell.give(500*charge_multiplier)
+		cell.give(25*charge_multiplier)
```

Comment:
This seems pretty extreme, 20 sheets for a full charge is a significant amount of plasma. Maybe something like 10 would be better

Reply:
I really would prefer to leave this as is. I am afraid that decreasing the price will not give the miners enough motivation to try something else as a secondary tool.

Reply:
as it stands this'll just kill killers completely lmao. Call it what it is.

Reply:
I'll copypaste my reply from another channel: 

>In an advanced cutter each ore piece lasts for 5 shots, 7 tiles each. That's 35 tiles. If you are searching for plasma ore, you will get, let's say, 3-4 plasma ore tiles, since plasma is common. each tile drops 1-5 ore pieces. Therefore per one ore in the field you will get on average ~7.5 ore pieces. ~14 if you go back and smelt the ore for fuel.

Reply:
It's not about the numbers. Nerfing something by making it more annoying is just really bad imo. You don't have to listen to me tho. I did offer an alternative idea but theres always the "do it yourself argument", so shrug.